feature/bgp/Import_Export_Policy/otg_tests/README.md

```diff
@@ -0,0 +1,216 @@
+# RT-1.64 BGP Import/Export Policy Functional Test Case
+
+## Objective
+
+To verify the basic functioanlity of BGP import and export policies and to check if they are correctly applied on a router.
+
+## Test Bed Setup
+Device Under Test (DUT): 1 x Router
+
+ATE BGP Peer: 1 x ATE with BGP Emulation capabilities.
+
+### Connections:
+One Ethernet link between a port on the DUT and a port on the ATE.
+Create the following connections:
+```mermaid
+graph LR; 
+subgraph ATE [ATE]
+    A1[Port1]
+end
+subgraph DUT1 [DUT1]
+    B1[Port1]
+end
+A1 <-- EBGP --> B1; 
+```
+DUT interface: IPV4 - 10.1.1.0/31 & IPV6 - 2001:db8:204:114::0/127
+ATE interface: IPV4 - 10.1.1.1/31 & IPV6 - 2001:db8:204:114::1/127
+
+## Prerequisites
+
+* Basic IPv4 and IPv6 connectivity configured between the DUT and ATE interface.
+* BGP peering established between the DUT and ATE on the connected interfaces.
+* Access to configure the DUT and ATE.
+* Understanding of BGP attributes (AS-Path, communities, local preference, MED).
+
+## Test Case Scenario
+
+This test case will focus on two scenarios:
+
+* Export Policy: Router exports only specific prefixes to ATE based on a policy.
+* Import Policy: Router imports only specific prefixes from ATE based on a policy.
+
+## Test Steps
+
+### Initial Setup (Pre-Test)
+
+### DUT Configuration (Basic BGP Peering):
+
+* Configure interface IP addresses (DUT: IPV4 - 10.1.1.0/31 & IPV6 - 2001:db8:204:114::0/127 , ATE: IPV4 - 10.1.1.1/31 & IPV6 - 2001:db8:204:114::1/127 ).
+* Enable BGP on the DUT.
+* Establish an EBGP peering session with the ATE (using ATE's AS as  AS65002).
+* Ensure the DUT has some local routes (loopback interfaces advertised into BGP) that can be filtered.
+
+
+### ATE Configuration (Basic BGP Emulation):
+
+* Configure the ATE port with an IP address (10.1.1.1/31) & IPv6 address (2001:db8:204:114::1/127).
+* Create a BGP Emulated Router on the ATE, acting as AS 65002.
+* Establish a BGP peering session with the DUT's IP (10.1.1.0) & IPV6 (2607:f8b0:8007:614f::).
+
+* Advertise multiple unique BGP routes from ATE, some intended to be filtered, and some to be allowed.
+  * Allowed routes: IPV4 192.0.2.1/32 (AS-Path: 65002), 192.0.2.2/32 (AS-Path: 65002) and IPV6 2001:db8:300:100::0/127 (AS-Path: 65002) , 2001:db8:300:101::0/127 (AS-Path: 65002)
+  * Denied routes:  IPV4 198.51.100.1/32 (AS-Path: 65002 65003), 198.51.100.2/32 (AS-Path: 65002 65004) and IPV6 2001:db8:400:100::1/128 (AS-Path: 65002 65004) , 2001:db8:400:101::1/128 (AS-Path: 65002 65004)
+
+### Verify BGP Peering:
+
+* On DUT: Check the BGP summary
+* On ATE: Check BGP session status in IxNetwork.
```

Comment:
We don't express these tests in terms of an implementation -- please again clarify what you want to check here.